db_access.py

```diff
@@ -1,21 +1,27 @@
 import sqlite3
-
+import datetime, hashlib
 from flask import app, g
 
 from chirpr import app
 
-def get_all_chirps():
+def hash_ps(text):
+    t = hashlib.sha224()
+    t.update(text.encode('utf-8'))
+    t = t.hexdigest()
+    return t
+
+
+def get_all_chirps(uid):
     conn = get_db()
-    return conn.execute('''
-        SELECT c.id, c.body, c.datetime, u.handle
-        FROM chirp c, user u
-        WHERE c.user_id = u.id
-    ''').fetchall()
+    chirps = []
+    for text in conn.execute('SELECT c.id, c.body, c.datetime, u.handle FROM chirp c, user u, followers f WHERE f.handleid = ? AND c.user_id = u.id AND c.user_id = f.followerid ORDER BY c.id', (uid,)):
+        chirps.append(text)
+    return reversed(chirps)
```

Comment:
I don't believe there is a `followers` table yet -- if you have made a DB migration for that, it didn't get checked in.

Also, don't use Python to reverse the order of the results from the DB. Write the query in a way to get the results in the order that you want. (Use `order by foo desc` if you want them backwards.

Finally, this is not related to creating new users, so remove it for now.